src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -514,9 +514,9 @@ std::wstring_view GlobalAppSettings::_SerializeTabWidthMode(const TabViewWidthMo
 }
 
 // Method Description:
-// - Adds the given colorscheme to our map of schemes, using its name as the key.
+// - Adds the given ColorScheme to our map of schemes, using its name as the key.

```

Comment:
I don't believe this needs to be capitalized as the object name. I think the generic "color scheme" terminology was sufficient.